### PR TITLE
Adds publish capabilities 

### PR DESCRIPTION
**Pull request by cannoneyed**
(to npm and as a standalone library via CDN)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,23 @@ const dataset = new Dataset(points);
...
+yarn install scatter-gl
+```
+
+##### via cdn
+
+```html
+<!-- Load three.js -->
+<script src="https://unpkg.com/three@0.106.2/build/three.min.js"></script>
+<!-- Load scatter-gl.js -->
+<script src="https://unpkg.com/scatter-gl@0.1.0/dist/scatter-gl.min.js"></script>
```
Comment (nsthorat) [inline README.md]: remove the /dist/scatter-gl.min.js

this should be taken care of by main

also if you care about stats use jsdelivr (unpkg doesnt give cdn hit counts, jsdelivr does)

File: README.md [hunk 2]
```diff
@@ -12,6 +12,23 @@ const dataset = new Dataset(points);
 const scatterGL = new ScatterGL(containerElement, dataset, params);
 ```
 
+## Installation
+
+##### with yarn / npm
+
+```bash
+yarn install scatter-gl
```
Comment (nsthorat) [inline README.md]: use yarn add

File: package.json [hunk 3]
```diff
@@ -10,11 +10,17 @@
   "repository": "PAIR-code/scatter-gl",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "unpkg": "lib/scatter-gl.min.js",
```
Comment (nsthorat) [inline package.json]: this should be dist to align with what you have in the README, same below

File: webpack/lib.config.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+const path = require('path');
```
Comment (nsthorat) [inline webpack/lib.config.ts]: add a license

File: README.md (hunk 1 above)
Comment (cannoneyed) [reply README.md]: main is gonna point to the non-bundled  index.js file for npm to use, so I'll point to the build lib/scatter-gl.min.js

File: README.md [hunk 5]
```diff
@@ -24,9 +24,9 @@ yarn install scatter-gl
 
 ```html
 <!-- Load three.js -->
-<script src="https://unpkg.com/three@0.106.2/build/three.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/three@0.106.2/build/three.min.js"></script>
 <!-- Load scatter-gl.js -->
-<script src="https://unpkg.com/scatter-gl@0.1.0/dist/scatter-gl.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/scatter-gl@0.1.0/lib/scatter-gl.min.js"></script>
```
Comment (nsthorat) [inline README.md]: sry I meant the jsdelivr field will do this for you, you don't need the stuff after 0.1.0

**Pull request merged**